readthedocs/restapi/utils.py

```diff
@@ -175,24 +174,16 @@ def index_search_request(
             'query': {
                 'bool': {
                     'must': [
-                        {
-                            'term': {
-                                'project': project.slug,
-                            },
-                        },
-                        {
-                            'term': {
-                                'version': version.slug,
-                            },
-                        },
+                        {'term': {'project': project.slug, }},
+                        {'term': {'version': version.slug, }},
                     ],
                     'must_not': {
                         'term': {
-                            'commit': commit,
-                        },
-                    },
-                },
-            },
+                            'commit': commit
```

Comment:
Please revert these formatting changes. Adding a trailing comma also makes future diffs clearer since we don't need to change the last item of a dictionary, list, or tuple:

```diff
# With a trailing comma:
foo = [
    "spam",
+   "eggs",
]

# Without a trailing comma:
foo = [
-   "spam"
+   "spam",
+   "eggs",
]
```